content/chainguard/chainguard-enforce/authentication/identity-examples/enforce-aws-identity.md

```diff
@@ -0,0 +1,342 @@
+images: []
+weight: 010
+---
+
+In Chainguard Enforce, [*assumable identities*](/chainguard/chainguard-enforce/iam-groups/assumable-ids/) are identities that can be assumed by external applications or workflows in order to perform certain tasks that would otherwise have to be done by a human.
+
+This procedural tutorial outlines how to create an identity using Terraform, and then create an AWS role that will assume the identity to interact with Chainguard resources. This can be used to authorize requests from AWS Lambda, ECS, EKS, or any other AWS service that supports [IAM roles for service accounts](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html).
+
+
+## Prerequisites
+
+To complete this guide, you will need the following.
+
+* `terraform` installed on your local machine. Terraform is an open-source Infrastructure as Code tool which this guide will use to create various cloud resources. Follow [the official Terraform documentation](https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli) for instructions on installing the tool.
+* `chainctl` — the Chainguard Enforce command line interface tool — installed on your local machine. Follow our guide on [How to Install `chainctl`](/chainguard/chainguard-enforce/how-to-install-chainctl/) to set this up.
+* An AWS account with the [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html) installed and configured. The [Terraform provider for AWS](https://registry.terraform.io/providers/hashicorp/aws/latest/docs) uses credentials configured via the AWS CLI.
+* A recent version of [Go](https://go.dev/dl/) to test the identity with AWS Lambda.
+
+## Creating Terraform Files
+
+We will be using Terraform to create an identity for an AWS role to assume. This step outlines how to create the Terraform configuration files that, together, will produce such an identity.
+
+These files are available here: https://github.com/chainguard-dev/platform-examples/tree/main/aws-auth
+
+To help explain each configuration file's purpose, we will go over what they do one by one. First, though, create a directory to hold the Terraform configuration and navigate into it.
+
+```sh
+mkdir ~/enforce-aws && cd $_
+```
+
+This will help make it easier to clean up your system at the end of this guide.
+
+### `main.tf`
+
+The first file, which we will call `main.tf`, will serve as the scaffolding for our Terraform infrastructure.
+
+The file will consist of the following content.
+
+```
+terraform {
+  required_providers {
+    aws        = { source = "hashicorp/aws" }
+    chainguard = { source = "chainguard-dev/chainguard" }
+    ko         = { source = "ko-build/ko" }
+  }
+}
+```
+
+This is a fairly barebones Terraform configuration file, but we will define the rest of the resources in the other two files. In `main.tf`, we declare and initialize the Chainguard Terraform provider.
+
+Next, you can look at the `chainguard.tf` file.
```

Comment:
```suggestion
Next, you can create the `chainguard.tf` file.
```

It's a bit pedantic but I try to avoid words like "look" or "see" in documentation to make it more inclusive for visually impaired folks